### PR TITLE
Fix scroll restoration

### PR DESCRIPTION
**Pull request by calebporzio**
Adjust scroll restoration so that it is the absolute LAST step in a page restoration (after Alpine initializes...)

### CODE REVIEW

**Comment by anhhh11** [comment]
degrade style not update?

File: dist/livewire.esm.js [hunk 1]
```diff
@@ -6816,13 +6820,17 @@ var require_module_cjs8 = __commonJS({
...
+            continue;
           if (!to.hasAttribute(name)) {
             from2.removeAttribute(name);
           }
         }
         for (let i = toAttributes.length - 1; i >= 0; i--) {
           let name = toAttributes[i].name;
           let value = toAttributes[i].value;
+          if (name === "style")
+            continue;
```
Comment (dauducmanh1812) [inline dist/livewire.esm.js:6833]: image
Degrade with inline styles

Comment (manavo) [reply dist/livewire.esm.js:6833]: @calebporzio I think I'm finding the same, that we now can't modify inline styles anymore, based on this.

**Comment by JayBizzle** [comment]
As others have said, this breaks updating `style` attributes :/

Totally broke a lot of our Livewire components

**Comment by manavo** [comment]
I think newer versions have fixed this, but haven't confirmed 100%. But I think I saw a commit this week or last week that should fix it

**Comment by manavo** [comment]
I think this commit, removing that check for "style": https://github.com/livewire/livewire/commit/774092003edb2670615ef09f3a9fbdd335d6d0d7

**Comment by JayBizzle** [comment]
> I think this commit, removing that check for "style": https://github.com/livewire/livewire/commit/774092003edb2670615ef09f3a9fbdd335d6d0d7

Thanks 👍🏻

**Pull request merged**